nservicebus/rabbitmq/connection-strings.md

```diff
@@ -18,7 +20,7 @@ RabbitMQ uses the [AMQP URI Specification](https://www.rabbitmq.com/uri-spec.htm
 
 ### Remote Host
 
-For remote host provide username and password because remote hosts don't accept default guest credentials
+By default, the [guest user can only connect via localhost](https://www.rabbitmq.com/access-control.html), so when connecting to a remote host, a user name and password must be provided.
```

Comment:
How about:

By default, the [guest user can only connect via localhost](https://www.rabbitmq.com/access-control.html). If connecting to a remote host, a user name and password must be provided.